justfile

```
@@ -21,6 +21,10 @@ fmt:
 format:
   cargo +$(cat ./nightly-version) fmt --all --check
 
+# Check the Docs and Doc tests
+doc:
+  RUSTDOCFLAGS="-A rustdoc::private-doc-tests" cargo +$(cat ./nightly-version) doc --workspace --all-features --no-deps
```

```suggestion
  RUSTDOCFLAGS="-D rustdoc::private-doc-tests, --cfg docsrs" cargo +$(cat ./nightly-version) doc --workspace --all-features --no-deps

```

This mimics the build command that docsrs uses, I've been meaning to add this to `justfile` for a while - thanks.